docs/guide/templates.md

```diff
@@ -155,9 +155,9 @@ In the simplified example above, when the `child.html.twig` template is used, th
 
 Now that you know how to create a general layout with Twig templates, let's take a look at the ways in which you can render content inside them.
 
-There are several ways of placing Content items or their Fields inside a template. You can do it using one of the [Twig functions described in detail here](twig_functions_reference.md).
+There are several ways of placing Content items or their Fields inside a template. You can do it using one of the [Twig functions described in detail here](content_rendering/twig_function_reference/twig_functions_reference.md).
 
-As an example, let's look at one of those functions: [ez\_render\_field](twig_functions_reference.md#ez_render_field). It renders one selected Field of the Content item. In its simplest form this function can look like this:
+As an example, let's look at one of those functions: [ez\_render\_field](content_rendering/twig_function_reference/field_twig_functions.md#ez_render_field). It renders one selected Field of the Content item. In its simplest form this function can look like this:
```

Comment:
```suggestion
As an example, look at the [ez\_render\_field](content_rendering/twig_function_reference/field_twig_functions.md#ez_render_field) function. 
It renders one selected Field of the Content item, and in the simplest form it can look like this:
```